json_prolog/launch/json_prolog.launch

```diff
@@ -1,4 +1,5 @@
 <launch>
-  <param name="initial_package" type="string" value="knowrob_common" />
+  <arg name="initial_package" default="knowrob_common" />
+  <param name="initial_package" type="string" value="$(arg initial_package)" />
```

Comment:
why did you changed this?

Reply:
It's an enhancement. It allows setting the initial package from the command line when executing the launch file.